{{cookiecutter.repostory_name}}/bin/backup-db-to-email.sh

```diff
@@ -1,17 +1,19 @@
-#!/bin/bash -eux
+#!/bin/bash -eu
 
 if [ "$(basename "$0")" == 'bin' ]; then
   cd ..
 fi
 
+if [ ! -e "$1" ]; then
```

Comment:
```suggestion
if [ ! -f "$1" ]; then
```
`-e` should only be used if you really don't care what it is (directory, file, symlink - you don't care). Here it needs to be a file.